contracts/core/EntryPoint.sol

```diff
@@ -316,8 +316,11 @@ contract EntryPoint is IEntryPoint, StakeManager {
             uint256 bal = balanceOf(sender);
             missingWalletFunds = bal > requiredPrefund ? 0 : requiredPrefund - bal;
         }
-        // solhint-disable-next-line no-empty-blocks
-        try IWallet(sender).validateUserOp{gas : mUserOp.verificationGasLimit}(op, opInfo.requestId, aggregator, missingWalletFunds) {
+        try IWallet(sender).validateUserOp{gas : mUserOp.verificationGasLimit}(op, opInfo.requestId, aggregator, missingWalletFunds) returns (uint256 deadline) {
```

Comment:
`deadline` is locally defined here (and in _validatePaymasterPrepayment) and not propagated up.  How will the bundler learn what the deadline is, in order to defend itself against short-lived ops and in order to drop expired ops efficiently without an additional simulation?  The simulation must return it and the bundler must be aware of it.